src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -106,28 +120,48 @@ export default {
 					}
 
 					// Check banner type
+
 					if (activePromoBanner.fields.bannerType === 'Appeal Banner') {
+						// Appeal Banner
 						this.appealEnabled = true;
 						this.appealBannerContent = activePromoBanner;
+					} else if (activePromoBanner.fields.bannerType === 'Custom Appeal') {
+						// Custom Banner
+						// Currently the only custom appeal banner is the 15 year appeal
+						this.appeal15Enabled = true;
+						this.appealBannerContent = activePromoBanner;
 					} else {
+						// Promo Banner
 						// parse the contentful richText into an html string
 						this.promoBannerContent = {
 							kvTrackEvent: activePromoBanner.fields.kvTrackEvent,
 							link: activePromoBanner.fields.link,
 							richText: documentToHtmlString(activePromoBanner.fields.richText),
 							iconKey: _get(activePromoBanner, 'fields.iconKey', 'present')
 						};
-
 						this.isPromoEnabled = true;
 					}
 				}
 			}
 		}
 	},
 	computed: {
+		show15YearAppeal() {
+			// only show the 15 year appeal on these pages
+			const whitelist = [
```

Comment:
Please add this to the Contentful custom data object for this banner. Initially I think the homepage might be turned off...

Reply:
Left code as is - If we want to initially hide on the homepage, '/' can be added to the 'Hidden URLs' property of the banner in contentful. I made this change on contentful dev